signaling-server/signaling_server.py

```diff
@@ -177,20 +180,37 @@ def _isAuthorized(user_password):
     return password is None or user_password == password
 
 
+class ExpandUserPath:
+    def __new__(cls, path: Union[str, Path]) -> Path:
+        return Path(path).expanduser().resolve()
```

Comment:
Automatically converts path parameters to absolute paths with user (`~`) expanded